### ISSUE TITLE
svg use tags also have href attributes

### ISSUE THREAD

**Issue by leroix** [kibu-australia/pushy#27]
It's common to use svg's in anchor tags with xlink'ing. Occassionally, a click will target the `use` tag, and since it has an `href` attribute, `recur-href` will stop there instead of iterating up to the anchor tag.

Shouldn't the `recur-href` function check whether the element is an anchor tag?

related: https://github.com/kibu-australia/pushy/issues/15

**Comment by livtanong** [comment]
Running into this issue as well. My urls are unexpectedly being set to `[object SVGAnimatedString]`.

**Comment by wavejumper** [comment]
@levitanong @leroix sorry for radio silence on this, updated `recur-href`logic - so this should work. 

Let me know if this solves your problem :)

**Comment by livtanong** [comment]
@wavejumper I saw that you replied to this thread (was notified via email), but it seems to be missing. You mentioned updating the logic, but I haven't seen any updates of this nature on master.

**Comment by wavejumper** [comment]
@levitanong Yeah, sorry about that! I only understood the nature of this problem after I made my comment. 

I'll merge in @leroix PR this weekend when I get the chance. Apologies for the delay

**Comment by livtanong** [comment]
Oh okay! Awesome. Thanks guys!

**Comment by bleakley** [comment]
`area` tags also have `href` attributes.
